package/ports_daemon/ports_daemon/src/ports.c

```diff
@@ -266,6 +282,30 @@ static port_config_t port_configs[] = {
     .restart = DO_NOT_RESTART,
     .first_start = true,
   },
+  {
```

Comment:
@benjaminaltieri We discussed making this a dev only change by disabling can0/can1 except when `/persistent/flags/enable_can` is `1`.   We also need to not add `can1` if we're on a `Duro`.

Reply:
I've added these follow up tasks to cover this:
  - https://swift-nav.atlassian.net/browse/DEVC-1246
  - https://swift-nav.atlassian.net/browse/DEVC-1247